PSBashCompletions/Get-MountPath.ps1

```diff
@@ -12,6 +12,7 @@
   # where it is mounted in bash. The resulting string should be either / or
   # /mnt/
   $found = $MountData | Where-Object { $_ -match "C:" -and $_ -match "(/mnt)?/c" }
+  $found = $found | Where-Object { $_ -match "(/mnt)?/c" } 
```

Comment:
Can this be one piped together? First "where" just matches C:, second "where" matches the mnt regex. Will that still work?

Reply:
Also, probably would be good to update the tests with another test that has your mount output to make sure this change both fixes the issue and doesn't break anything.

Reply:
Oh! My mistake (on a phone, trying to respond fast) - I am guessing a "select -first 1" at the end and then just a one line -match to get the matches corrected would work. The thing I'm hung up on is the re-match being in a where clause - it's already filtered, so the "where" is confusing. There's no way in six months I'll remember why it was re-filtered. :)

Reply:
I figured out the bug occurring! :) i have a second mount that starts with `C:` but does not contain `(/mnt)?/c` at all (it is a mount created by docker. Will be updating the pull request in a little bit with my `bash -c mount` output as a test case.

Also, changing the whole match to something wayyy simpler I just thought of while bouncing it off someone else. Why not instead of answering "is it (/mnt/c or /c) or /?" (trying to see if it uses drive letters) just boil it down to "does it use /mnt/ or / prefix?" with a match for `"/mnt/[a-z] "` and then it will both match against any drive letter system not relying C: drive and will allow for returning the constants `"/mnt/"` or `"/"` without relying on Matches in the first place. also then able to match it against `"on /mnt/[a-z] " to more closely follow the mount output for hopefully less headaches in the future.

Reply:
Nice! I'm AFK for a couple days but will merge and release this as soon as I'm back!